src/brsTypes/components/RoSGNode.ts

```diff
@@ -16,6 +16,8 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
     readonly kind = ValueKind.Object;
     elements = new Map<string, BrsType>();
     private fields = new Map<string, Field>();
+    private children = new Array<RoSGNode>();
```

Comment:
Mind switching this to an array literal?  I don't think `new Array<Type>` is being used anywhere else.  Perhaps I should add a lint check for that 😄 

```suggestion
    private children: RoSGNode[] = [];
```